manuscripts2/README.md

```diff
@@ -1,20 +1,123 @@
 # Readme
 
-This folder is an initialization to the new functions and classes that have to be added to manuscripts. This file provides a brief introduction to these classes and functions.
+## Instructions and description about manuscripts2
 
+### How do I generate a report from this?
 
-### TODO:
-- Is popping from dict and then creating the nested aggregation the most efficient way to do this?
-- How do you parse 3 level nested aggregation? 
-- Create a Metrics class which takes the data from the Query class, processes it and gives us an output?
+Here it is:
+First of all, unlike the previous version of manuscripts, this version has two different data sources for GitHub data. One is `github_issues` and the other one is `github_prs`. The `github_issues` data source contains data related to both, the issues and the prs found in the repo. How ever, each item in the data source is treated as an issue i.e the items of the **pull requests** categories don't have the pull requests data. The `github_prs` data source, on the other hand, only has pull requests and the data related to them. (github_issues data source will have more items than github_prs data source).
 
-### Classes:
+These data sources have their own ES indices from which they query data from. This helps us separate the issues and PRs in a github repository.
+
+You can produce the github_issues index using the following command:
+```bash
+p2o.py --enrich --index github_issues_raw --index-enrich github_issues -e http://localhost:9200 --no_inc --debug --db-host localhost --db-sortinghat <db name> --db-user root github <owner> <repo name> -t <token>
+```
+This will create an raw index by the name of `github_issues_raw` containing the issues+prs raw data from \<repo name\>.
+The enriched data will be stored in the `github_issues` index.
+
+And a github_prs index using this command:
+```bash
+p2o.py --enrich --index github_prs_raw --index-enrich github_prs -e http://localhost:9200 --no_inc --debug --db-host localhost --db-sortinghat <db name> --db-user root github <owner> <repo name> -t <token> --category pull_request
+```
+This will create an raw index by the name of `github_prs_raw` containing the PR only raw data from \<repo name\>.
+The enriched data will be stored in the `github_prs` index.
+
+The git index has no changes and can be produced as follows:
+```bash
+p2o.py --enrich --index git_raw --index-enrich git -e http://localhost:9200 --no_inc --debug --db-host localhost --db-sortinghat <db name> --db-user root git <repository url>
+```
+
+**NOTE:** If you don't know how or what [Sortinghat](https://github.com/chaoss/grimoirelab-sortinghat) is, please [go through the tutorial](https://grimoirelab.gitbooks.io/tutorial/sortinghat/intro.html). Sortinghat helps us manage the identities of the contributors and manitainers.
```

Comment:
Add something like "But in this case, you don't really need to know about it if you don't need identity management, which for a first try, likely you don't need".